src/api/routes/auth.js

```diff
@@ -60,4 +62,39 @@ export default (app) => {
             return next(err);
         }
     });
+
+    router.post("/recover/request", validators.recover, async (req, res, next) => {
+        try {
+            const accountService = new AccountService();
+
+            const account = await accountService.findByEmail(req.body.email);
+
+            if (account === null) {
+                return res.status(HTTPStatus.OK).json({});
+            }
+
+            const link = accountService.buildPasswordRecoveryLink(account);
+            accountService.sendPasswordRecoveryNotification(account, link);
+
+            return res.status(HTTPStatus.OK).json({});
+        } catch (err) {
+            console.error(err);
+            return next(err);
+        }
+    });
+
+    router.get("/recover/:token/confirm", validators.confirmRecover, validToken, (req, res) => res.status(HTTPStatus.OK).json({}));
```

Comment:
Can you explain this further?

Reply:
I imagine this route being used by the frontend to evaluate if the page can be accessed by the user